infrastructure/shifty.bicep

```diff
@@ -0,0 +1,41 @@
+param location string = resourceGroup().location
```

Comment:
Opinion. We probably could move this bicep code to the main bicep file as there is not much going on here, so probably no need to have it in its own file

Reply:
I kind of like having them separated, but I don't have a strong opinion - however, moving it into the same file creates some funky stuff with the target scope. Currently the target scope specified in the main azuredeploy file is "subscription", while the target of Shifty is the shifty resource group. If we put them into a single file, Bicep complains about the scopes being different